### ISSUE TITLE
Adding a sum column ?

### ISSUE THREAD

**Issue by Floflinou** [FriendsOfSylius/SyliusImportExportPlugin#225]
Hi,

Would it be possible to add a "Total" row (ie. Sum of Tax Amount ...) to the exported grid ? If so, how could it be achieved ?

Thanks !

**Comment by oallain** [comment]
Hi @Floflinou 

You need to had it for Orders?
You can override fields exported here https://github.com/FriendsOfSylius/SyliusImportExportPlugin/blob/a87f312c00f6645d2c8f29ee21b164e8907d2586/src/Resources/config/services_export_csv.yml#L23

Regards

**Comment by Floflinou** [comment]
Hi @ollain,

Actually, I've already done the override. What I'm trying to do is to calculate the sum of x values, as in an excel function: "=SUM(Fx:Fy)" where x = first row and y = last row.

I hope it's a little clearer !

**Comment by t-n-y** [comment]
Hi,

Have you found a solution to do that ? have to do the same and i didn't found a solution already

**Comment by Floflinou** [comment]
Hi,

I did, I created new functions in the corresponding Controller classes which filled new fields. These fields were also added to the plug-in parameters file.

Essentially, find the places where the fields are defined and filled, and create "totalXfield" methods where necessary. 

I'm really sorry if I'm being vague, it's been a long time and I switched company so I don't have access to what i coded anymore...

Hope you'll be able to figure it out !
